### ISSUE TITLE
The TabStrip tablist lacks a tabindex attribute

### ISSUE THREAD

**Issue by aleksandarevangelatov** [telerik/kendo-ui-core#7272]
### Bug report

The TabStrip tablist has a `role=tablist` attribute, but lacks a tabindex attribute

### Reproduction of the problem
Run the [Kendo UI TabStrip Demo](https://demos.telerik.com/kendo-ui/tabstrip/keyboard-navigation) and inspect the markup. 

### Environment

* **Kendo UI version:** 2023.1.314
* **Browser:** [all ]
